bazel/README.md

```diff
@@ -1,12 +1,23 @@
 # Bazel
 
-## Alternative Official Image
+The `gcr.io/cloud-builders/bazel` image is maintained by the Cloud Build team,
+but it may not support the most recent features or versions of Bazel. We also do
+not provide historical pinned versions of bazel.
 
-A supported `bazel` image, including multiple tagged versions,
-is available at
-http://gcr.io/cloud-marketplace-containers/google/bazel.
+A supported `bazel` image, including multiple tagged versions, is maintained by
```

Comment:
I'd be weary of saying "supported" here. Maybe:

> The Bazel team provides an image at ... that supports multiple tagged versions.

Reply:
Throughout

Reply:
Missed this -- it came in just after PR was closed.
https://github.com/GoogleCloudPlatform/cloud-builders/pull/685